packages/flutter_tools/lib/src/test/flutter_web_platform.dart

```diff
@@ -618,7 +618,7 @@ class BrowserManager {
       throwToolExit('${runtime.name} exited with code $browserExitCode before connecting.');
     }).catchError((dynamic error, StackTrace stackTrace) {
       if (completer.isCompleted) {
-        return;
+        return null;
```

Comment:
You have the same changes in your previous PR https://github.com/flutter/flutter/pull/72409

Reply:
Oops yikes I must have branched from my branch.